README.md

```diff
@@ -1,8 +1,6 @@
 # GOV.UK Prototype kit
 
-## News
-
-**Upgrading from version 1 to 2:** the latest version of the kit (2.0.0 and later) is not compatible with previous versions. If you update your old prototypes you'll need to [convert them as well](https://govuk-prototype-kit.herokuapp.com/docs/updating-the-kit).
+Go to the [GOV.UK Prototype Kit site](https://govuk-prototype-kit.herokuapp.com/docs) to download the latest version and read the documentation.
 
 ## About the prototype kit
```

Comment:
I don't personally see the relevance to linking to them here. But I don't feel strongly about it.